package/MDAnalysis/core/groups.py

```diff
@@ -2531,21 +2531,31 @@ def improper(self):
         return topologyobjects.ImproperDihedral(self.ix, self.universe)
 
     def write(self, filename=None, file_format="PDB",
-              filenamefmt="{trjname}_{frame}", **kwargs):
+              filenamefmt="{trjname}_{frame}", frames=None, **kwargs):
         """Write `AtomGroup` to a file.
 
         The output can either be a coordinate file or a selection, depending on
-        the `format`. Only single-frame coordinate files are supported. If you
-        need to write out a trajectory, see :mod:`MDAnalysis.coordinates`.
+        the format. 
+
+        Examples
+        --------
+
+        >>> ag = u.atoms
+        >>> ag.write('selection.ndx')  # Write a gromacs index file
+        >>> ag.write('coordinates.pdb')  # Write the current frame as PDB
+        >>> # Write the trajectory in XTC format
+        >>> ag.write('trajectory.xtc', frames='all')
+        >>> # Write every other frame of the trajectory in PBD format
+        >>> ag.abswrite('trajectory.pdb', frames=u.trajectory[::2]
```

Comment:
misses trailing `)`

Reply:
`abswrite` ????

Reply:
Can you add an example with `multiframe`, too, please?

Reply:
... and, yay, examples! 👍

Reply:
Failing autocomplete + late coding = such ridiculous typo -_-